### PR TITLE
Add change_url script, remove helpers & sudo prefixes (YunoHost 2.7.2)

### PR DESCRIPTION
**Pull request by JimboJoe**
## Problem
Add a change_url script for `yunohost app change-url` feature.

## Solution
- Change the way the nginx.conf is adapted at install/upgrade: some lines were removed when installed on "/" path --> they must be kept as comments to allow switching to a "/path" path
- Add a change_url script: as the nginx.conf file is not simple, the code can't be generic and symetric --> every case (/ -->  /path, /path --> /, /path --> /path2) is differentiated
- Depend now on YunoHost 2.7.2 to allow some cleaning: remove sudo prefixes and integrated  helpers

These have been splitted in 3 different commits to ease the review process.

## PR Status
Work finished. Package_check, basic tests and upgrade from last version OK.
Could be reviewed and tested.

## Validation

- [ ] ~**Complete test** :~
- [ ] ~**Upgrade previous version** :~
- [x] ~**Code review** : Maniack C~
- [x] ~**Approval (LGTM)** : Maniack C~
- [ ] ~**Approval (LGTM)** :~
- [x] ~**CI succeeded** : Maniack C - Level 7 with change_url.~

Since this PR has been rerouted from master to testing. It passes to a minor decision.
- [x] **Upgrade previous version** : Maniack C
- [x] **Code review** : Maniack C
- [x] **Approval (LGTM)** : Maniack C
- [x] **Approval (LGTM)** : 
- [x] **CI succeeded** : Maniack C - Level 7 with change_url.
- [x] **CI succeeded** : [Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/wallabag2_ynh%20enh_add_change_url%20(Official)/)  
When the PR is mark as ready to merge, you have to wait for 3 days before really merge it.

### CODE REVIEW

**Comment by JimboJoe** [comment]
Note: I opened [bug #999](https://dev.yunohost.org/issues/999) regarding changing path from "/path" to "/" ("//" final path).

File: check_process [hunk 1]
```diff
@@ -15,6 +15,7 @@
 		multi_instance=1
 		incorrect_path=1
 		port_already_use=0
+    change_url=1
```
[the remark this answers is not in the document]
Comment (JimboJoe) [reply check_process]: I guess everything else was OK for you...?

**Comment by maniackcrudelis** [comment]
Upgrade OK

Will be merged in 3 days.

**Comment by lapineige** [comment]
Upgrade working on my side too.
Changing the url from /path1 to /path2 works too. Changing the main domain works too.
(not tested with /->/path and /path->/)

**Pull request merged**